docs/introduction/computational_resources_overview.md

```diff
@@ -1,47 +1,34 @@
-Elastic Machine Groups with dynamic scaling, and MPI Clusters for complex simulations 
-requiring multiple CPUs. Lastly, we present a comparison using a SWASH simulation 
-to demonstrate the performance differences between shared and dedicated resources, 
-providing practical insights into optimizing your simulations with the Inductiva API.
+it comes to making informed decisions about resource allocation. Here, you will 
+learn the two options available for running your simulations; shared and dedicated
+resources. Additionally, we will demonstrate the performance differences between 
+the two resources using a SWASH simulation.
 
 ## Shared Resources
 
 As a standard practice, when you submit simulation tasks through the Inductiva API, 
-they get initially sent to a shared pool of workers on designated Virtual Machines (VMs)
-serving multiple users. These VMs are specifically allocated to facilitate easy 
+they enter a [task queue]() and are sent to a shared pool of workers on designated Virtual Machines (VMs)
+serving multiple users from either the [Google Cloud Provider (GCP)](https://cloud.google.com/compute/docs/machine-resource) or Inductiva's own 
+computational platform (ICE). These VMs are specifically allocated to facilitate easy 
 API testing and to handle light tasks efficiently with a minimal setup, ideal for 
 quick experimentation.
 
-Tasks enter a [task queue]() and are allocated to workers from either the [Google Cloud Provider (GCP)](https://cloud.google.com/compute/docs/machine-resource) or Inductiva's own 
-computational platform (ICE), ensuring a streamlined start for for doing quick 
-experimentation with the Inductiva API.
-
 However, this shared resource pool has its limitations, including slower completion 
-times for simulations due to its finite capacity and less powerful VMs. This setup, 
-while user-friendly for initial tests and small-scale simulations, may not suffice 
-for more demanding tasks, which require more robust computational power and shorter 
-waiting times.
-
-For running a larger volume of simulations that require more powerful VMs, the 
-Inductiva API provides you with the option to set up your own [dedicated resources]() 
-to ensure you can access the necessary computational power for your simulations 
-without the constraints of a shared environment. 
+times for simulations due to its finite capacity and less powerful VMs. For running 
+a larger volume of simulations, the Inductiva API provides you with the option to 
+set up your own [dedicated resources](). 
 
 ## Dedicated Resources
 
-Inductiva provides the capability to create dedicated pools of VM resources, 
-termed [Machine Groups](), exclusively reserved for your use and not shared with 
-others. 
+To ensure you can access the necessary computational power for your simulations 
+without the constraints of a shared environment, Inductiva provides the option 
+to create dedicated pools of VM resources, termed [Machine Groups](), exclusively 
```

Comment:
remove `termed [Machine Groups]()` since there are several options and mentioned them already below.